src/braket/aws/aws_session.py

```diff
@@ -197,15 +202,14 @@ def _add_braket_user_agents_handler(self, request: awsrequest.AWSRequest, **kwar
             request.headers.add_header("User-Agent", self._braket_user_agents)
 
     @staticmethod
-    def _add_cost_tracker_count_handler(request: awsrequest.AWSRequest, **kwargs) -> None:
+    def _add_cost_tracker_count_handler(request: awsrequest.AWSRequest, **kwargs) -> None:  # noqa: ARG004
```

Comment:
A lot of our functions take kwargs for no apparent reason and it drives up the complexity

Reply:
I agree. I want these revisited separately due to this complexity. I think the repo should adopt a better posture around using these and documentation.